src/Contracts/Auth0Repository.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Marketredesign\MrdAuth0Laravel\Contracts;
+
+use Exception;
+
+interface Auth0Repository
```

Comment:
Would introduce a breaking change though, since it's also possible to dependency inject these repositories. Can be included in a future new major version.

Reply:
Let's do that.